storybook/src/elements/nuxeo-checkmark/nuxeo-checkmark.stories.js

```diff
@@ -0,0 +1,20 @@
+import { storiesOf } from '@storybook/polymer';
+import { boolean, color } from '@storybook/addon-knobs';
+import { html } from 'lit-html';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-checkmark';
+
+storiesOf('UI/nuxeo-checkmark', module).add('nuxeo-checkmark', () => {
+  const bgColor = color('--nuxeo-checkmark-background-color', '#fffff', 'CSS variables');
+  const bgColorChecked = color('--nuxeo-checkmark-background-color-checked', '#0000ff', 'CSS variables');
+  const checked = boolean('checked', false);
+  const disabled = boolean('disabled', false);
+  return html`
+    <style>
+      * {
+        --nuxeo-checkmark-background-color: ${bgColor};
+        --nuxeo-checkmark-background-color-checked: ${bgColorChecked};
+      }
+    </style>
+    <nuxeo-checkmark ?checked="${checked}" ?disabled="${disabled}"></nuxeo-checkmark>
```

Comment:
UX nit: Looking at this element again, I'm wondering if we should rethink the way the element is presented when it is both checked and disabled. It looks like it is not disabled and therefore interactable. Maybe we can create a ticket to improve it.

Reply:
yeah it def needs an ux/ui revamp ,  created a ticket for that https://jira.nuxeo.com/browse/ELEMENTS-1145